src/components/ErrorShow/ErrorShow.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+import './ErrorHide.scss';
```

Comment:
The imported SCSS file name should be 'ErrorShow.scss' instead of 'ErrorHide.scss' to match the component name.